common/src/utilities.rs

```diff
@@ -17,3 +19,11 @@ pub fn read_ron<T>(ron_path: & std::path::Path) -> std::result::Result<T, Box<dy
     de.end()?;
     Ok(deserialized)
 }
+
+pub fn set_rotation_toward_vector(transform: &mut Transform, vector: &Vector2<f32>) {
+    let mut angle = Vector2::new(0.0, -1.0).angle(vector);
```

Comment:
Just for the sake of correctness, what happens if the vector is very close to (0.0, 0.0) I think it should be covered by a test. (In the collision systems when the centers are on the same point I deliberately changed a small but not zero direction to avoid infinite jump because of a collision).
Or maybe some kind of error should be returned if we cannot "fake" a proper behaviour